test/nistrng/rng.c

```diff
@@ -0,0 +1,102 @@
+//
+//  rng.c
+//
+//  Created by Bassham, Lawrence E (Fed) on 8/29/17.
+//  Copyright © 2017 Bassham, Lawrence E (Fed). All rights reserved.
+//  Modified for liboqs by Douglas Stebila
+//
```

Comment:
I do not like this one at all. 
We'll have to replace that. We are working on that for https://github.com/pq-code-package/mlkem-c-embedded and I will get back to you once it's done so you can plug it in here.

Reply:
We have a reimplemented version now here: https://github.com/pq-code-package/mlkem-c-embedded/pull/30 
Once that is merged you can use it here instead of the one you currently have.

Reply:
done in https://github.com/pq-code-package/mlkem-c-aarch64/pull/29/commits/e7d6d8d4f7982aab1db7cda447ca69addb287d52